### ISSUE TITLE
Soviet Harvester stops and does not continue to work

### ISSUE THREAD

**Issue by CCCP84** [CnCNet/cncnet-yr-client-package#387]
After all available gold has been collected, the Soviet Harvester stops and will not continue working until it is forced to start again.
the alliance harvester does not have this problem. After all the gold is collected it also stops but when the drill extracts new gold the harvester automatically starts working. 
We play long games on a map with islands and soviet players are forced to constantly control all islands every 5 minutes during a long game and resume mining this is a very routine and stupid work in addition players forget to control this during the battle and simply lose money because if the Harvester does not mine gold it does not accumulate due to a bad algorithm and is simply lost.
This is a very annoying problem that occurs constantly, players are tired of it.
I hope the solution to this problem is not very complicated and it will be solved.

**Comment by alexp8** [comment]
not aware of any fix for this

**Comment by CCCP84** [comment]
But why you closed it? It is still a problem. Maybe somebody sometimes will be able to fix it?
Maybe it possible to look, how allied miner starts mining when ore appears and do the same way with soviet?
This problem is really annoying..

**Comment by alexp8** [comment]
Yeah this isn't a code change we can make in yr-client-package that could fix this, a phobos expert like Kerb or ZivDero may have an idea

I believe it to be a spawner update

**Comment by CCCP84** [comment]
Solved in Phobos:
https://github.com/Phobos-developers/Phobos/pull/2157
